nix/tahoe-lafs.nix

```diff
@@ -1,7 +1,7 @@
 { python2Packages }:
 let
   repo = ((import ./tahoe-lafs-repo.nix) + "/nix");
-  tahoe-lafs-drv = repo + "/default.nix";
+  tahoe-lafs-drv = repo + "/tahoe-lafs.nix";
```

Comment:
I was mildly concerned about expanding the public interface of the Tahoe-LAFS Nix packaging from `default.nix` to `tahoe-lafs.nix` and `overlays.nix` but since everything is pinned I guess at worst we have to deal with upgrade challenges at some point in the future but not any surprise breakage.